code/modules/client/preference/preferences.dm

```diff
@@ -2035,6 +2038,10 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 
 	character.add_language(language)
 
+	if(species == "Grey")
+		if(speciesprefs)
+			character.speech_span = "wingdings"
```

Comment:
A more complicated change would be necessary - this is the only place you should ever transfer preferences to a character. You'd have to attach it to something persistent, like the mind or DNA. In this case, probably the mind.